fameal/ape-tax
# Feature/logo align

## What it does ✨
Align the logos and title of the vaults.
This pr depends on #79.  
This alignment is built for two "elements" (simple/complex emojis or alphanumeric characters), such as `🏆🚀`, `🏅💪` or `₿👋` and aims to visually align the logo and title between all lines.  
It's not perfect because : 
- Emojis have different sizes.
- We can't specify a specific width per letter/emoji in a simple way.
- We have to handle logos with more than 2 characters differently.
- We have to do some specific manipulations to get the real emoji (sometimes an emoji can have 6 characters, that is a group of several emojis). 

**But it works**

## How to test ✅
Install the dependencies with `npm install` and start the app with `npm run serve` and check for the alignment.
